autotest.pm

```diff
@@ -147,13 +142,13 @@ sub runalltests {
                 }
             }
             else {
-                if ($snapshots_supported && $flags->{milestone}) {
+                if ($snapshots_supported && ($flags->{milestone} || $bmwqemu::vars{MAKETESTSNAPSHOTS})) {
                     make_snapshot('lastgood');
```

Comment:
Right now I'm merely questioning the usefulness of the original implementation. What is the use case of having snapshots for each test module?
When I develop new tests I only needed to quickly get to the latest failed test module to fix it. (This brings another question whether the rollback for non fatal tests at L:149 should be allowed)

Ultimately I see that I will introduce another variable to preserve the original functionality and introduce mine.